test/server/reporters-test.js

```diff
@@ -336,4 +352,12 @@ describe('Reporters', function () {
     it('xunit', function (done) {
         testReporter('xunit', done);
     });
+
+    it('Should xunit reporter use plain text decorator', function () {
+        testUsingPlainTextDecorator('xunit');
+    });
+
+    it('Should json reporter use plain text decorator', function () {
```

Comment:
*Should xunit and json reporters use the plain text decorator*
or, if we want a really thoughtful one,
*Should reporters that target machine-readable formats always use the plain text decorator*